fastlane_bot/events/exceptions.py

```diff
@@ -4,4 +4,11 @@ def __init__(self, filepath):
 
     def __str__(self):
         return (f"tokens.csv does not exist at {self.filepath}. Please run the bot without the `read_only` flag to "
-                f"create this file.")
\ No newline at end of file
+                f"create this file.")
+
+
+class AyncUpdateRetryException(Exception):
```

Comment:
Looks like there's a missing "s" at the beginning of this class name.

Reply:
This is a custom exception class